app/src/main/kotlin/com/softteco/template/navigation/BottomNavigationItem.kt

```diff
@@ -36,4 +38,11 @@ sealed class BottomNavigationItem(
         selectedIcon = Icons.Filled.Settings,
         route = Screen.Settings.route
     )
+
+    object Navigation : BottomNavigationItem(
```

Comment:
The implication was that we would use the home screen for the main feature. I don't think it makes sense to keep a blank Home and add the feature as a separate route

Reply:
The navigation functionality will be in a separate branch. I think that the navigation application will be built only for customers for whom this particular functionality will be important or basic. That's why I added it to the bottom menu, but I can remove it if you insist :)

Reply:
The idea was to place custom functionality on the Home screen. This way, the applications in these “feature branches” will look more functional (Onboarding, Main feature, Profile, Setting; without skeleton screen). Of course, it's up to you)